dashboard/streamlit_app.py

```diff
@@ -77,6 +81,7 @@ def get_circuit_selection(conn: connect, key: str) -> st.multiselect:
     judge_selection = st.multiselect(key=key,
                                      placeholder="select circuit(s)",
                                      options=rows,
+                                     default=None,
```

Comment:
What does this improve?

Reply:
i think this ensures that the multiselect initialises without any selection, so users would start with an untouched multiselect